crates/solvers/src/domain/solution.rs

```diff
@@ -13,6 +13,16 @@ pub struct Solution {
 /// addresses to price in an arbitrarily denominated price.
 pub struct ClearingPrices(pub HashMap<eth::TokenAddress, U256>);
 
+macro_rules! clearingprices {
+    ($($args:tt)*) => {
+        $crate::domain::solution::ClearingPrices(
+            ::maplit::hashmap!($($args)*),
+        )
+    };
+}
+
+pub(crate) use clearingprices;
```

Comment:
We have done this in other places but I agree that I don't like it. I don't even use the `maplit::hashmap!` anymore since you can create HashMaps from array iterators like

```rust
let a: HashMap<int, int> = [(0, 1), (2, 3)].into_iter().collect();
```

Reply:
Used @vkgnosis suggested iterator method.

3653111eec2e68781a96275781348c66c4182d35